src/sentry/notifications/notifications/rules.py

```diff
@@ -146,6 +151,14 @@ def get_image_url(self) -> str | None:
             return image_builder.get_image_url()
         return None
 
+    def is_new_design(self) -> bool:
```

Comment:
I will clean this up after removing feature flags. Right now 3 feature flags are involved which makes the logic complicated.